bin/ch-test

```diff
@@ -560,28 +577,32 @@ test_make_template_print () {
     return
 }
 
-pack_fmt_set () {
-    width=$1
-    if [[ -n $pack_fmt ]]; then
-        case $pack_fmt in
-        '🐘')  # elephant emoji
-            pack_fmt=squash
-            ;;
-        '📠')  # fax machine emoji
-            pack_fmt=tar
-            ;;
-        esac
-        export CH_PACK_FMT=$pack_fmt
-        method='command line'
-    elif [[ -n $CH_PACK_FMT ]]; then
-        method='environment'
-    else
-        pack_fmt_choose
-        method='default'
+test_one_file () {
```

Comment:
Could we force the scope to full in this function? It feels silly to have to specify full scope to test an example not in standard scope. Could we call `require_unset` on scope and just set it to full?

Reply:
I guess you could easily just set your scope to full with an environment variable if you wanted to be lazy. It could get confusing for people running full scope tests when you didn't mean to.